website/description_issues.qmd

```diff
@@ -106,3 +106,29 @@ It is also not recommended to [mention package names](https://cran.r-project.org
 
 ------------------------------------------------------------------------
 
+# Using Authors@R
+
+
+## Problem
+
+You have not properly declared the authors, maintainers and copyright holders in the Authors@R section of the DESCRIPTION file.
+
+## Solution
+
+Add the `Authors@R` field in the DESCRIPTION file. 
+
+
+### Details
+
+You can find more roles in the help documentation by running `?utils::person`. Make sure the provided email address is actively monitored, as it will be the primary point of contact with CRAN for future updates and fixes.
+
+```{r}
+Authors@R: person("Jane", "Smith", email = "chef@cran-cookbook.com",
+  role = c("aut", "cre"))
```

Comment:
```suggestion
  role = c("aut", "cre"), ORCID = "<USER-ORCID>")
```
Related to the commends about ORCID, it was recommended to add this identifier: it helps detecting people using different emails and might help to track down people for messages. 
Related to this there is also now a recommendation for companies/organizations to use their own identifiers ROR, I think.

Reply:
Yes! Forgot to mention those identifiers. Thanks for the comment